### ISSUE TITLE
$eq shortcut nested inside $or not supported

### ISSUE THREAD

**Issue by nolanlawson** [pouchdb/pouchdb#6261]
_From @jo on October 18, 2016 15:24_

This does not work in pouchdb-find, while it works against the http adapater in CouchDB 2.0:

```
$or: [
    {"name.first": "Nancy"},
    {"name.first": "Mick"}
]
```

See https://github.com/jo/pouchdb-find/commit/46e4454324749bf5e97f73027cd29814f7d983f0 for failing test.

_Copied from original issue: nolanlawson/pouchdb-find#218_

**Comment by nolanlawson** [comment]
I don't believe $or is supported. There should be an open issue but I didn't find one.

**Comment by nolanlawson** [comment]
_From @BigBlueHat on February 3, 2017 21:32_

Actually, it does work (afaict), you just have to explicitly use `$eq`:

```
$or: [
    {"name.first": {"$eq": "Nancy"}},
    {"name.first": {"$eq": "Mick"}}
]
```

According to the readme `$or` was added in 0.6.0.

**Comment by nolanlawson** [comment]
_From @BigBlueHat on February 6, 2017 18:53_

Actually. It helps to read. This issue is precisely about `$eq` being required here... Thanks for reporting it @jo. Sorry for the noise...

**Comment by afinne** [comment]
Duplicate of of #6366 and fixed in #6368
